x-pack/plugins/ml/server/routes/modules.ts

```diff
@@ -118,9 +138,108 @@ export function dataRecognizer({ router, mlLicense }: RouteInitialization) {
    *
    * @api {get} /api/ml/modules/get_module/:moduleId Get module
    * @apiName GetModule
-   * @apiDescription Returns module by id.
-   *
-   * @apiParam {String} [moduleId] Module id
+   * @apiDescription Retrieve a whole ML module, containing jobs, datafeeds and saved objects.
+   * @apiParam {String} [moduleId] ID of the module to return. If no module ID is supplied,
+   *      an array of all modules will be returned.
```

Comment:
you can extract 
```
schema.object({
          ...getModuleIdParamSchema(true),
        })
```
to an export module in `x-pack/plugins/ml/server/routes/schemas` and use `@apiSchema` annotation instead

Reply:
Done in https://github.com/elastic/kibana/pull/66738/commits/7a6fd145038f2be82eebe24681815f7ca03d38cc